samples/System.IO.Pipelines.Samples/CompressionSample.cs

```diff
@@ -4,8 +4,6 @@
 
```

Comment:
Note that this sample was removed from the solution as it no longer builds (we removed all the System.IO.Pipelines.* projects). What do we plan to do with this?

cc @davidfowl, @pakrym